task/dev8/src/com/github/zubarevladimir/Matrix/MatrixMultiply.java

```diff
@@ -0,0 +1,58 @@
+package com.github.zubarevladimir.Matrix;
+
+import com.sun.javaws.exceptions.InvalidArgumentException;
+
+/**
+ * Contains method for multiply matrix.
+ */
+public class MatrixMultiply {
+
+  /**
+   * Multiply given matrix.
+   *
+   * @param matrixOne array contains first matrix elements.
+   * @param matrixTwo array contains second matrix elements.
+   * @return double[][] - result matrix.
+   * @throws InvalidArgumentException throw if matrix can't multiply.
+   */
+  public double[][] doMultiply(double[][] matrixOne, double[][] matrixTwo)
+      throws InvalidArgumentException {
+
+    int linesOne = matrixOne.length;
+    int columnsOne = matrixOne[0].length;
+    int linesTwo = matrixTwo.length;
+    int columnsTwo = matrixTwo[0].length;
+
+    if (columnsOne != linesTwo) {
+      String[] argumentsArray = {"Columns first matrix: " + Integer.toString(columnsOne),
+          "Lines second matrix: " + Integer.toString(linesTwo)};
+      throw new InvalidArgumentException(argumentsArray);
```

Comment:
invalid using of InvalidArgumentException . You must specify which exactly argument of method is invalide